pages/css/facts.css

```diff
@@ -0,0 +1,75 @@
+* {
+    border: 1px solid black;
+    font-family: 'Roboto', sans-serif;
+    /* margin: 0 auto; */
+    text-align: center;
+}
+
+
+
+h1 {
+    font-size: 30px;
+    /* text-align: center; */
+    background-color: lightskyblue;
+    color: white;
+}
+
+nav {
+    text-align: center;
+    background-color: aqua;
+}
+
+nav ul {
+    /* display: inline-block;  this should go to li*/
+    list-style: none;
+    background-color: bisque;
+}
+
+
+nav ul li {
+    display: inline-block;
+    font-size: 25px;
+    width: 25%;
+    background-color: pink;
+    margin: auto;
+}
```

Comment:
Nice work using this descendant selector, typically we name folders that hold css files "styles"